src/index.html

```diff
@@ -6,22 +6,328 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Catalog</title>
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    />
     <link
       rel="preconnect"
-      href="https://fonts.gstatic.com"
+      href="https://fonts.googleapis.com"
     />
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
-      rel="stylesheet"
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
     />
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
       rel="stylesheet"
-      href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list_element">
+            <a
+              href="#apple"
+              class="header__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#samsung"
+              class="header__link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#smartphones"
+              class="header__link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#laptop&computers"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#gadgets"
+              class="header__link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#tablets"
+              class="header__link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#photo"
+              class="header__link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="header__list_element">
+            <a
+              href="#video"
+              class="header__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
```

Comment:
The repeated 'card' blocks should be generated using a loop in a templating language or in JavaScript, rather than hardcoding each block. This would make the code DRY and easier to maintain.